project-api/project-api-test/src/test/resources/myproject_with_lang_import/Ballerina.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+org = "sameera"
+name = "myproject"
+version = "0.1.0"
```

Comment:
Add new line at end of file

Reply:
added in the 274babad95104c166f871220329483954e88cf7b commit